### ISSUE TITLE
Fine-tuning on VQA-RAD & SLAKE datasets

### ISSUE THREAD

**Issue by o0t1ng0o** [xiaoman-zhang/PMC-VQA#1]
Hi @xiaoman-zhang,

Thank you for sharing this code of your interesting work PMC-VQA.
I found that you reported the performance on VQA-RAD and SLAKE dataset.
Could you provide the details about the finetuning on VQA-RAD and SLAKE dataset?
Thank you in advance.

Best,

**Comment by xiaoman-zhang** [comment]
We have updated the finetune dataset and code in ./src/MedVInT_TE/

**Comment by avi-jit** [comment]
Thanks @xiaoman-zhang for the code updates! Could you also please add or summarize the evaluation code for the two datasets using ACC metric as described in the paper?
